nautobot/extras/models/customfields.py

```diff
@@ -398,29 +392,19 @@ class CustomField(BaseModel, ChangeLoggedModel, NotesMixin):
     ]
 
     class Meta:
-        ordering = ["weight", "name"]
+        ordering = ["weight", "label"]
```

Comment:
Should this be `["weight", "key"]` instead?

Reply:
I feel like it would make more sense to use `label` here since the __str__() method is returning `self.label` but I could go either way do not have a super strong opinion here.

Reply:
I was thinking it would match the sort order on the object's `_custom_field_data`. Probably not a big deal either way

Reply:
I guess this https://github.com/nautobot/nautobot/actions/runs/4443594213/jobs/7801045894?pr=3426 is the sequence of code to execute if you want that error to pop out.